meta-mender-tegra/recipes-bsp/u-boot/libubootenv-fake/fw_printenv

```diff
@@ -9,26 +18,34 @@ if [ "$1" = "-n" ]; then
     fi
 fi
 if [ -z "$1" ]; then
-    echo "mender_boot_part=`nvbootctrl get-current-slot`"
+    bootpart=`get_bootpart`
+    echo "mender_boot_part=$bootpart"
     exit 0
 fi
 while [ -n "$1" ]; do
     case "$1" in
-        mender_boot_part|mender_boot_part_hex)
+        mender_boot_part)
+            bootpart=`get_bootpart`
             [ -n "$quiet" ] || echo -n "$1="
-            nvbootctrl get-current-slot
+            echo "$bootpart"
+            ;;
+        mender_boot_part_hex)
+            bootpart=`get_bootpart`
+            bootpart_hex=`echo 16o${bootpart}p | dc`
+            [ -n "$quiet" ] || echo -n "$1="
+            echo "$bootpart_hex"
             ;;
         mender_uboot_separator)
             [ -n "$quiet" ] || echo -n "$1="
             echo "something other than just 1"
             ;;
         upgrade_available)
-            [ -n "$quiet" ] || echo -n "$1="
-	    if [ -e "/var/lib/mender/upgrade_available" ]; then
-		echo "1"
-	    else
-		echo "0"
-	    fi
+            [ -n "$quiet" ] || echo "$1="
```

Comment:
@nielsavonds I didn't catch this when I reviewed the change but I just noticed there's no longer a `-n` here, which means  I see :
```
root@jetson-tx2:~# fw_printenv upgrade_available
upgrade_available=
1
```
instead of 
```
root@jetson-tx2:~# fw_printenv mender_boot_part
mender_boot_part=33
```
Is this on purpose?

Reply:
@madisongh do you agree I should add the `-n` back in `upgrade_available`?  I'd like to get this PR upstreamed.

Reply:
Yes... sorry I missed that in the earlier review.

Reply:
No worries, I missed it too.  Fixed in https://github.com/OE4T/meta-mender-community/commit/43c00e2066c5526d9732c515bb64bc0dcbac4911, will get this pushed up.